From 747e72a0f5034c4ae09f9cd32d8a198beef0b01e Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Fri, 15 Sep 2017 14:25:06 +0800
Subject: [PATCH 718/895] soc: ti: wkupm3: pick up the leftover from Merge to
 fix this compiling error

drivers/soc/ti/wkup_m3_ipc.c: In function 'wkup_m3_copy_aux_data':
drivers/soc/ti/wkup_m3_ipc.c:112:18: error: too few arguments to function 'rproc_da_to_va'
  aux_data_addr = rproc_da_to_va(m3_ipc->rproc,
                  ^~~~~~~~~~~~~~
In file included from drivers/soc/ti/wkup_m3_ipc.c:29:0:
./include/linux/remoteproc.h:567:7: note: declared here
 void *rproc_da_to_va(struct rproc *rproc, u64 da, int len, u32 flags);

Signed-off-by: Zumeng Chen <zumeng.chen@windrivder.com>
---
 drivers/soc/ti/wkup_m3_ipc.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/soc/ti/wkup_m3_ipc.c b/drivers/soc/ti/wkup_m3_ipc.c
index a90ddf8..c9a9c37 100644
--- a/drivers/soc/ti/wkup_m3_ipc.c
+++ b/drivers/soc/ti/wkup_m3_ipc.c
@@ -109,9 +109,9 @@ static unsigned long wkup_m3_copy_aux_data(struct wkup_m3_ipc *m3_ipc,
 	void *aux_data_addr;
 
 	aux_data_dev_addr = WKUP_M3_DMEM_START + WKUP_M3_AUXDATA_OFFSET;
-	aux_data_addr = rproc_da_to_va(m3_ipc->rproc,
-				       aux_data_dev_addr,
-				       WKUP_M3_AUXDATA_SIZE);
+	aux_data_addr = rproc_da_to_va(m3_ipc->rproc, aux_data_dev_addr,
+			WKUP_M3_AUXDATA_SIZE, RPROC_FLAGS_NONE);
+
 	memcpy(aux_data_addr, data, sz);
 
 	return WKUP_M3_AUXDATA_OFFSET;
-- 
1.7.5.4

